### PR TITLE
Improve Dependabot automerge automation

### PR DESCRIPTION
**Pull request by estern1011**
Updates our Dependabot configuration to use the new Github Native Dependabot (the old one doesn't work anymore!)
Based on: https://github.com/panorama-ed/base-dependabot-automation

### CODE REVIEW

**Comment by cyrorama** [comment]
It looks like we need to update `.rubocop.yml`:
```
inherit_gem:
  panolint: rubocop.yml
```

It should instead be:
```
inherit_gem:
  panolint: panolint-rubocop.yml
```

According to https://github.com/panorama-ed/panolint

**Review by cyrorama — APPROVED** [review]
LGTM

File: order_as_specified.gemspec [hunk 1]
```diff
@@ -17,7 +17,6 @@ Gem::Specification.new do |spec|
 
   spec.files         = `git ls-files -z`.split("\x0")
   spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
-  spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
```
Comment (estern1011) [inline order_as_specified.gemspec:20]: Fixes `order_as_specified.gemspec:20:3: C: [Correctable] Gemspec/DeprecatedAttributeAssignment: Do not set test_files in gemspec.`

File: order_as_specified.gemspec [hunk 2]
```diff
@@ -32,4 +31,5 @@ Gem::Specification.new do |spec|
   # explicitly specify the version here
   sqlite3 = ENV["ACTIVERECORD_VERSION"] == "~> 5.0.0" ? "~> 1.3.13" : "~> 1.4"
   spec.add_development_dependency "sqlite3", sqlite3
+  spec.metadata["rubygems_mfa_required"] = "true"
```
Comment (estern1011) [inline order_as_specified.gemspec:34]: `Gemspec/RequireMFA: metadata['rubygems_mfa_required'] must be set to 'true'.`

File: .github/workflows/tests.yml [hunk 3]
```diff
@@ -8,7 +8,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ ubuntu-latest ]
-        ruby: [ 2.5, 2.6, 2.7 ]
```
Comment (estern1011) [inline .github/workflows/tests.yml:11]: Only ruby 2.5 was failing, don't think this version is still necessary?

**Pull request merged**